tests/testthat/test-combiners.R

```diff
@@ -0,0 +1,12 @@
+context("Combine task targets into summary files or objects")
+
+test_that("combine_to_tibble works on all reasonable numbers of files", {
+  tfiles <- tempfile(pattern=as.character(1:3))
+  for(i in 1:3) readr::write_lines(i, path=tfiles[i])
```

Comment:
was hoping you'd catch that the files would need to be different for this test :)